core/action.class.inc.php

```diff
@@ -168,6 +170,54 @@ protected function DoCheckIfHasTrigger()
 			$this->m_aCheckWarnings[] = Dict::S('Action:WarningNoTriggerLinked');
 		}
 	}
+
+	/**
+	 * @since 3.2.0 N°5472 method creation
+	 */
+	public function DisplayBareRelations(WebPage $oPage, $bEditMode = false)
+	{
+		parent::DisplayBareRelations($oPage, false);
+
+		if ($oPage instanceof iTopWebPage) {
+			$this->GenerateLastExecutionsTab($oPage, $bEditMode);
+		}
+	}
+
+	/**
+	 * @since 3.2.0 N°5472 method creation
+	 */
+	protected function GenerateLastExecutionsTab(iTopWebPage $oPage, $bEditMode)
+	{
+		$oRouter = Router::GetInstance();
+		$sActionLastExecutionsPageUrl = $oRouter->GenerateUrl('notifications.action.last_executions_tab', ['actionid' => $this->GetKey()]);
+		$oPage->AddAjaxTab('action_errors', $sActionLastExecutionsPageUrl, false, Dict::S('Action:last_executions_tab'));
+	}
+
+	/**
+	 * @throws InvalidConfigParamException
+	 * @since 3.2.0 N°5472 method creation
+	 */
+	public function GetLastExecutionsTabContent(WebPage $oPage): void
+	{
+		$oConfig = utils::GetConfig();
+		$sLastExecutionDaysConfigParamName = 'notifications.last_executions_days';
+		$iLastExecutionDays = $oConfig->Get($sLastExecutionDaysConfigParamName);
+
+		if ($iLastExecutionDays <= 0) {
+			throw new InvalidConfigParamException("Invalid value in the {$sLastExecutionDaysConfigParamName} config parameter");
+		}
```

Comment:
```suggestion
		if ($iLastExecutionDays < 0) {
			throw new InvalidConfigParamException("{$sLastExecutionDaysConfigParamName} config. parameter must be a positive integer");
		}
```

In functional review, we discussed that 0 would mean no time limit.
You'll need to adjust the OQL as well.

Reply:
I've changed the exception message with 5b1ff2c3
I used the param description to avoid duplicate. 
We might create a dedicated method in the Config class to check value... But it's out of scope of this PR.

Reply:
Woops sorry correct commit is c5f6c47c

Reply:
0 as config param value : cdb5556a